### PR TITLE
update comment in aptos_containers

### PR DESCRIPTION
**Pull request by andrew2003**
# Check List

- [x] Don't forget to squash commits into meaningful chunks before merging
- [x] Check every test passed.
- [x] Did you split imports into std, third parties, and custom parts?
- [x] Format your code.
  - [x] cargo fmt
  - [x] cargo clippy
  - [x] `taplo fmt --config taplo/taplo.toml`
- [x] Check unwrap, expect().
- [x] Check clone().
- [x] Check your commit messages.
- [x] Have you added meaningful comments.

### CODE REVIEW

File: src/aptos_container.rs [hunk 1]
```diff
@@ -113,47 +160,16 @@ impl AptosContainer {
...
-            let guard = self.accounts_channel_tx.read().await;
-            for account in accounts {
-                guard.as_ref().unwrap().send(account).await?;
-            }
-        }
-        result
-    }
-
+    /// Get `accounts` from `override_accounts` in `AptosContainer` if override_accounts
+    /// is `Some`. If `None` call to `lazy_init_accounts` to init and return `accounts`.
```
Comment (zk-steve) [inline src/aptos_container.rs]: Rewrite this

File: src/aptos_container.rs [hunk 2]
```diff
@@ -163,56 +179,211 @@ impl AptosContainer {
...
+    /// # Example
+    /// ```rust
+    /// use aptos_testcontainer::aptos_container::AptosContainer;
+    ///
+    /// #[tokio::main]
+    /// async fn main() {
+    ///     let aptos_container = AptosContainer::init().await.unwrap();
+    ///     let command = "bin/sh -c mkdir my_file".to_string();
+    ///     let (_, stderr) = aptos_container.run_command(&command).await.unwrap();
+    ///     println!("stderr: {:?}", stderr)
```
Comment (zk-steve) [inline src/aptos_container.rs]: should print both stdout and stderr

File: src/aptos_container.rs [hunk 3]
```diff
@@ -163,56 +179,211 @@ impl AptosContainer {
...
+    /// ```rust
+    /// use aptos_testcontainer::aptos_container::AptosContainer;
+    ///
+    /// #[tokio::main]
+    /// async fn main() {
+    ///     let aptos_containe = AptosContainer::init().await.unwrap();
+    ///     let accounts = aptos_containe.lazy_init_accounts().await.unwrap();
+    /// }
+    /// ```
     pub async fn lazy_init_accounts(&self) -> Result<()> {
```
Comment (zk-steve) [inline src/aptos_container.rs]: make this function private

File: src/aptos_container.rs [hunk 4]
```diff
@@ -163,56 +179,211 @@ impl AptosContainer {
...
         *self.accounts_channel_rx.lock().await = Some(rx);
-
+        // Assign the sender to accounts_channel_tx to finalize the initialization.
         *guard = Some(tx);
+        // Return success.
         Ok(())
     }
 
+    /// Copies contract files from a local directory into the container's filesystem. The files
+    /// are base64-encoded and transferred in chunks to avoid issues with large files.
```
Comment (zk-steve) [inline src/aptos_container.rs]: Only explain what the function does; don't explain how it does it.

File: src/aptos_container.rs [hunk 5]
```diff
@@ -237,27 +408,147 @@ impl AptosContainer {
...
+                guard.as_ref().unwrap().send(account).await?;
+            }
+        }
+        result
+    }
+
+    /// Executes a script located within the specified directory.
+    ///
+    /// # Parameters
+    /// - `local_dir`: The directory path containing the scripts to be executed.
```
Comment (zk-steve) [inline src/aptos_container.rs]: The directory path containing contract code

File: src/aptos_container.rs [hunk 6]
```diff
@@ -298,24 +627,29 @@ impl AptosContainer {
...
         }
-
+        // Copy contracts to a new location and log the time taken.
         let now = Instant::now();
         let contract_path = self.copy_contracts(local_dir).await?;
         debug!("copy_contracts takes: {:.2?}", now.elapsed());
 
         let contract_path_str = contract_path.to_str().unwrap();
 
+        // Override `Move.toml` if no sub-packages are provided.-
```
Comment (zk-steve) [inline src/aptos_container.rs]:
```suggestion
        // Override `Move.toml` if no sub-packages are provided.
```

File: src/utils.rs [hunk 7]
```diff
@@ -0,0 +1,23 @@
+use tiny_keccak::Hasher;
+
+pub fn get_account_address(private_key: &str) -> String {
```
Comment (zk-steve) [inline src/utils.rs:26]: Doc for this function

File: Cargo.toml [hunk 8]
```diff
@@ -12,15 +12,15 @@ rand = "0.8.5"
 regex = "1.10.6"
 serde = { version = "1.0" }
 testcontainers = { version = "0.21.1" }
+tiny-keccak = { version = "2.0.2", features = ["sha3"] }
 thiserror = { version = "1.0.63" }
 tokio = { version = "1.40.0", features = ["rt", "rt-multi-thread", "macros"] }
 walkdir = "2.3.3"
+hex = "0.4.3"
+ed25519-dalek = { version = "2.1.1" }
```
Comment (zk-steve) [inline Cargo.toml]: taplo

File: src/aptos_container.rs (hunk 1 above)
Comment (andrew2003) [reply src/aptos_container.rs]: done

File: src/aptos_container.rs (hunk 2 above)
Comment (andrew2003) [reply src/aptos_container.rs]: done

File: src/aptos_container.rs (hunk 3 above)
Comment (andrew2003) [reply src/aptos_container.rs]: done

File: src/aptos_container.rs (hunk 4 above)
Comment (andrew2003) [reply src/aptos_container.rs]: Done

File: src/aptos_container.rs (hunk 5 above)
Comment (andrew2003) [reply src/aptos_container.rs]: done

File: README.md [hunk 9]
```diff
@@ -6,11 +6,131 @@
 
 ## Introduction
 
-Test container for Aptos Node
+This module provides a simple and smart test container framework for testing interactions with an Aptos node.
 
 ## Install
+Add this to your `Cargo.toml`:
```
Comment (zk-steve) [inline README.md]:
```suggestion

Add this to your `Cargo.toml`:
```

File: README.md [hunk 10]
```diff
@@ -6,11 +6,131 @@
...
+
+```toml
+[dependencies]
+aptos-testcontainer = { git = "https://github.com/sota-zk-labs/aptos-testcontainer.git" }
+```
+## Usage
+Here’s how to use the `aptos-testcontainer` in your tests:
```
Comment (zk-steve) [inline README.md:12]:
```suggestion

Here’s how to use the `aptos-testcontainer` in your tests:
```

**Review by zk-steve — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 11]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - master
+    tags:
```
Comment (zk-steve) [inline .github/workflows/ci.yaml]: separate flow

**Review by zk-steve — APPROVED** [review]

**Pull request merged**